app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkViewModelTest.kt

```diff
@@ -14,6 +14,7 @@ import org.junit.Before
 import org.junit.Test
 import org.mozilla.fenix.TestUtils
 import org.mozilla.fenix.TestUtils.bus
+import org.mozilla.fenix.ext.minus
```

Comment:
Nit: we can remove this import maybe?

Reply:
It's an extension operator. We'd have to add it to the class to remove the import.